docs/src/Box.doc.js

```diff
@@ -795,22 +859,21 @@ card(
   <MainSection name="Related">
     <MainSection.Subsection
       description={`
-    **[Flex](/Flex)**
-    Use Flex for flexbox layouts, especially when even spacing between elements is desired, by using the \`gap\` property.
+        **[Flex](/Flex)**
+        Use Flex for flexbox layouts, especially when even spacing between elements is desired, by using the \`gap\` property.
 
-    **[Container](/Container)**
-    Use Container to responsively layout content with a max-width on large screens.
+        **[Container](/Container)**
+        Use Container to responsively layout content with a max-width on large screens.
 
-    **[ScrollableContainer](/ScrollableContainer)**
-    For proper positioning when using anchor components (Flyout, Tooltip, etc.) that can scroll within the viewport, use a ScrollableContainer.
+        **[ScrollableContainer](/ScrollableContainer)**
+        For proper positioning when using anchored components (Flyout, Tooltip, etc.) that can scroll within the viewport, use a ScrollableContainer.
```

Comment:
`that can scroll within the viewport` >> `located within a scrolling container.`